src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/LanguageServices/LanguageServiceHost.cs

```diff
@@ -146,47 +146,48 @@ private Task ExecuteWithinLockAsync(Func<Task> task)
         /// Ensures that <see cref="_currentAggregateProjectContext"/> is updated for the latest target frameworks from the project properties
         /// and returns this value.
         /// </summary>
-        private Task<AggregateWorkspaceProjectContext> UpdateProjectContextAsync()
+        private Task<AggregateWorkspaceProjectContext?> UpdateProjectContextAsync()
         {
             // Ensure that only single thread is attempting to create a project context.
-            AggregateWorkspaceProjectContext previousContextToDispose = null;
+            AggregateWorkspaceProjectContext? previousContextToDispose = null;
             return ExecuteWithinLockAsync(async () =>
             {
                 await _commonServices.ThreadingService.SwitchToUIThread();
 
-                string newTargetFramework = null;
+                string? newTargetFramework = null;
                 ConfigurationGeneral projectProperties = await _commonServices.ActiveConfiguredProjectProperties.GetConfigurationGeneralPropertiesAsync();
 
                 // Check if we have already computed the project context.
-                if (_currentAggregateProjectContext != null)
+                AggregateWorkspaceProjectContext? currentAggregateProjectContext = _currentAggregateProjectContext;
```

Comment:
This is a workaround for https://github.com/dotnet/roslyn/issues/31620 which I'm still hitting with Preview 1 at home